CHANGELOG.md

```diff
@@ -2,14 +2,10 @@
 
 ## 2.13.0
 
-### Misc. Updates
+### Bug Fixes
 
 - **Breaking Change** Updated nullability flags for the `AWSSignatureSignerUtility`, `AWSSignatureV4Signer`, `AWSSignatureV2Signer` and `AWSS3ChunkedEncodingInputStream` classes. Methods in these classes will have new signatures in Swift code, and may require code changes in your app to consume. 
```

Comment:
While you're in here, please add a link to the PR & Issue that tracks this change.